crates/bevy_internal/src/default_plugins.rs

```diff
@@ -35,6 +35,11 @@ impl PluginGroup for DefaultPlugins {
         group.add(bevy_input::InputPlugin::default());
         group.add(bevy_window::WindowPlugin::default());
 
+        #[cfg(debug_assertions)]
```

Comment:
I know it's not optimal to use `debug_assertions`, but the user can still disable/enable the plugin independently of the `debug_assertions` using the `App::add_plugins_with`. I'd welcome any improvement suggestions, though.